### ISSUE TITLE
Search fails with 500 CORS error

### ISSUE THREAD

**Issue by cgreene** [AlexsLemonade/refinebio#883]
### Context

Searching for anything on refine.bio in chrome or firefox fails.

### Problem or idea

Console says: 
```
Access to fetch at 'https://api.refine.bio/search/?limit=10&offset=0' from origin 'https://www.refine.bio' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```

### Solution or next step

The API should be allowing requests from our frontend.

### New Issue Checklist

- [x] The title is short and descriptive.
- [x] You have explained the context that led you to write this issue.
- [x] You have reported a problem or idea.
- [x] You have proposed a solution or next step.
